src/main/scala/com/thirdship/libunit/CompoundTSUnit.scala

```diff
@@ -165,17 +165,22 @@ class CompoundTSUnit(val numerator: List[TSUnit] = List.empty[TSUnit],
 		}
 	}
 
+	private def keySetEquals(unit: CompoundTSUnit): Boolean = {
+		numeratorDimensions.keySet.equals(unit.numeratorDimensions.keySet) &&
```

Comment:
Do we need this? It worries me that we have it, yet it is not a pure function. That could lead to mis use in the future.